Makefile

```diff
@@ -20,10 +20,7 @@ dev-app:
 	cd services/agora && yarn dev 
 
 dev-api:
-	cd services/api && pnpm start:dev 
+	cd services/api && && docker-compose up -d && pnpm start:dev  
```

Comment:
We shouldn't do this, because `docker compose up -d` to run the database can be done only once, and then the database runs continuously in the background, whereas we restart the backend all the time.
Also, the `docker-compose.yml` for starting the postgresql database is there as a useful courtesy but some developers prefer using other techniques to run their database!